test/pages/sections/guarantor/GuarantorRequiredPageSpec.scala

```diff
@@ -31,7 +32,7 @@ import play.api.test.FakeRequest
 
 class GuarantorRequiredPageSpec extends SpecBase {
 
```

Comment:
Could probably do with another test or 2 to make sure the `isRequired` function works properly